common/src/main/java/com/microsoft/identity/common/adal/internal/cache/StorageHelper.java

```diff
@@ -522,11 +522,12 @@
     }
 
     private boolean logIfKeyHasChanged(@NonNull final SecretKey hmacKey) throws NoSuchAlgorithmException, NoSuchPaddingException, InvalidKeyException, BadPaddingException, IllegalBlockSizeException {
+        final String methodName = ":logIfKeyHasChanged";
```

Comment:
you can also do, (here and other places)
final String methodTag = TAG + ":logIfKeyHasChanged";
this way you dont need to do <TAG + method name > in each of the log statement.

Reply:
Here: da1e62d1ddacd758eb5d0a6cedd9589521475ab8